### ISSUE TITLE
Using --icon when packaging

### ISSUE THREAD

**Issue by cosmin-novac** [arboleya/electrify#58]
I tried different paths for the icon.png file but none seems to work and I can't find any documentation. What is the correct path to use? Also, is it relative to the main folder or the .electrify folder? Or an absolute path?

**Comment by ghost** [comment]
It seems that elctrify can only set icon from *.ico file. At least on 
windows. I do use --icon parameter with following option:

electrify package -o c:\dist2 -- --icon=.electrify/icon_48_full.ico

  and run it from project root.

**Comment by derwok** [comment]
Yes. On **MacOS** is has to be an *.icns file. Trying a *.png file silently ignored my "--icon" parameter.
On MacOSX this worked for me:

`electrify package -- --icon=/path/to/logo/appicon.icns`

Maybe the electrify readme doc could clarify things up here?
